cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -223,8 +228,13 @@ func (o Options) run(stopCh <-chan struct{}) error {
 		return err
 	}
 
+	protobufConfig := *kubeAPIServerConfig
+	if protobufConfig.ContentType == "" {
+		protobufConfig.ContentType = runtime.ContentTypeProtobuf
```

Comment:
`ContentType` indicates only the content type of the request body that clients sends to the server. What about the `AcceptContentTypes` (the content type of response that the server sends to the clients) - what is its default value? `application/json`?

Reply:
According to the [doc string](https://github.com/kubernetes/client-go/blob/ab82d40f6e857a3162e22ac8a5888b6314f9b0eb/rest/config.go#L282-L284) `AcceptContentTypes` is defaulted to `ContentType`.
My reasoning was, it should be enough to default `ContentType` because it will also be used for `AcceptContentTypes` if the operator left the field empty. If the operator specified a value, the operator probably knows what they are doing and the value should be respected.

Reply:
I checked the details now:
The `Accept` header is set to `AcceptContentTypes` if specified, otherwise set to `ContentType + ", */*"`, see [here](AcceptContentTypes).

Reply:
Ok, thanks for explaining.